scripts/bg-selector.js

```diff
@@ -51,8 +51,8 @@ export default (function ($, EventDispatcher) {
 
     // Labels for radio buttons
     var radioLabels = [
-      H5PEditor.t('H5PEditor.NDLACoursePresentation', 'setImageBackground', {}),
-      H5PEditor.t('H5PEditor.NDLACoursePresentation', 'setColorFillBackground', {})
+      t('setImageBackground', {}),
```

Comment:
This will break, because `t` is not imported in `bg-selector.js`.

Reply:
Yet another point in favour of typing 🥲